Solar-Wrapper/src/TrackablesSolARImpl.cpp

```diff
@@ -203,6 +200,88 @@ namespace org::openapitools::server::implem
+        SolAR::datastructure::Transform3Df localCRS(matrix);
+
+        //unitsystem
+        SolAR::datastructure::UnitSystem unitSystem = resolveUnitSystem(trackable.getUnit());
+
+        //dimension
+        Eigen::Vector3d dimension = Eigen::Vector3d(trackable.getTrackableSize().data());
+
+        //taglist
+        std::multimap<std::string,std::string> keyvalueTagList;
+        for (const auto &tag : trackable.getKeyvalueTags()){
+            for(const auto &value : tag.second){
+                keyvalueTagList.insert({tag.first,value});
+            }
+        }
+
+        //trackable type
+        SolAR::datastructure::StorageTrackableType type = SolAR::datastructure::resolveTrackableType(trackable.getTrackableType());
+
+        //encoding info
+        SolAR::datastructure::EncodingInfo encodingInfo(trackable.getTrackableEncodingInformation().getDataFormat(), trackable.getTrackableEncodingInformation().getVersion());
+
+        //payload
+        std::vector<std::byte> payload = TrackablesSolARImpl::to_bytes(trackable.getTrackablePayload());
+
+        //name
+        std::string name = trackable.getName();
+
+        //id
+        boost::uuids::uuid id = xpcf::toUUID(trackable.getUUID());
+
+        //create the trackable
+        SRef<SolAR::datastructure::StorageTrackable> storageTrackable = xpcf::utils::make_shared<SolAR::datastructure::StorageTrackable>(id, creatorId, localCRS, unitSystem, dimension, keyvalueTagList, type, encodingInfo, payload, name);
+
+        //adding the newly created StorageTrackable to the worldgraph
+        xpcf::uuids::uuid trackableId;
+        switch(m_worldStorage->modifyTrackable(trackableId, storageTrackable))
+        {
+            case SolAR::FrameworkReturnCode::_SUCCESS :
+            {
+                //initialize the json object that we will send back to the client (the trackable's id)
+                std::string trackableIdString = xpcf::uuids::to_string(trackableId);
+                auto jsonObjects = nlohmann::json::array();
+                nlohmann::to_json(jsonObjects, trackableIdString);
+
+                //send the ID to the client
+                response.headers().add<Pistache::Http::Header::ContentType>(MIME(Application, Json));
+                response.send(Pistache::Http::Code::Ok, jsonObjects.dump());
+
+                break;
+            }
+
+            case SolAR::FrameworkReturnCode::_NOT_FOUND :
+            {
+                response.headers().add<Pistache::Http::Header::ContentType>(MIME(Text, Plain));
+                response.send(Pistache::Http::Code::Not_Found, "Trackable not found\n");
+
+                break;
+            }
+
+            default :
+            {
+            //if something went wrong
+            response.headers().add<Pistache::Http::Header::ContentType>(MIME(Text, Plain));
+            response.send(Pistache::Http::Code::Internal_Server_Error, "something went wrong when modifying the trackable to the world storage");
```

Comment:
You added `\n` in the previous message: if it is required, add one here as well, otherwise remove both ?